src/index.js

```diff
@@ -1,24 +1,17 @@
 import React from "react";
 import ReactDOM from "react-dom";
+import thunk from "redux-thunk";
+import { applyMiddleware, createStore } from "redux";import { Provider } from "react-redux";
+import App from './App'
+import Reducer from "./reducers";
```

Comment:
you'll more commonly see reducers names in lowercase if its one word like here or camelCase if its 2 words, not required, but its not very commonly seen capitalized